handler/CMakeLists.txt

```diff
@@ -44,33 +44,41 @@ if (ANDROID)
     )
 endif (ANDROID)
 
-add_executable(handler main.cc)
-target_sources(handler PRIVATE ${CRASHPAD_HANDLER_LIBRARY_FILES})
-target_compile_features(handler PUBLIC cxx_std_17)
-target_include_directories(handler PRIVATE ..)
-target_link_libraries(handler PRIVATE client tools compat snapshot minidump)
+if (HANDLER STREQUAL "INTEGRATED")
+    add_library(handler STATIC ${CRASHPAD_HANDLER_LIBRARY_FILES})
+    target_compile_features(handler PUBLIC cxx_std_17)
+    target_link_libraries(handler PRIVATE client tools compat snapshot minidump)
+    target_include_directories(handler PUBLIC ..)
+else ()
+    add_executable(handler main.cc)
+    set_property(TARGET handler PROPERTY ENABLE_EXPORTS 1)
```

Comment:
Shouldn't need to specify `ENABLE_EXPORTS` if a separate target is being generated for the static library.